packages/server/src/index.ts

```diff
@@ -122,7 +122,11 @@ export class App {
             const username = process.env.FLOWISE_USERNAME
             const password = process.env.FLOWISE_PASSWORD
             const basicAuthMiddleware = basicAuth({
-                users: { [username]: password }
+                users: { [username]: password },
+                authorizer: (username: string, password: string) => {
+                    this.authenticated = username === process.env.FLOWISE_USERNAME && password === process.env.FLOWISE_PASSWORD
```

Comment:
not recommend to do `===` comparsion, use `safeCompare` - https://www.npmjs.com/package/express-basic-auth#custom-authorization